subscription/add-seats.md

```diff
@@ -6,15 +6,15 @@ redirect_from:
 - /docker-hub/billing/add-seats/
 ---
 
-You add seats anytime to your existing subscription.
+You can add seats anytime to your existing subscription.
 
-## Add seats to a monthly subscription
+## Add seats to a subscription
```

Comment:
Wondering if we can remove this heading here as it looks like a repetition?

![image](https://user-images.githubusercontent.com/47779042/134325569-926b503d-ff99-4c29-ad00-06bc2368bdd8.png)